Library/Homebrew/cmd/audit.rb

```diff
@@ -218,6 +218,27 @@ def audit_urls
       problem "Google Code homepage should end with a slash (URL is #{homepage})."
     end
 
+    # Automatic re-direct exists, but this is another hugely common error.
```

Comment:
There's an unnecessary ` - ` in this sentence. I'll kill it when I incorporate whatever feedback maintainers give later.